relay-metrics/src/protocol.rs

```diff
@@ -349,20 +419,27 @@ fn parse_tags(string: &str) -> Option<BTreeMap<String, String>> {
 
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
 pub struct Metric {
-    /// The name of the metric without its unit.
+    /// The name of the metric, i.e. the MRI.
     ///
-    /// Metric names cannot be empty, must start with a letter and can consist of ASCII
-    /// alphanumerics, underscores, slashes, @s and periods.
-    pub name: String,
-    /// The unit of the metric value.
+    /// The metric name historically has been a freeform string. Over time we realized that we can
```

Comment:
All of this is still an experimental feature, so we do not have to comment much on obsolete "historical" states. Note that all of this has been described in https://github.com/getsentry/relay/pull/1215 (description) and you can reuse much text from there if you'd like to give more motivation.

Reply:
ok sure, I copied that wording. I don't think historical context is irrelevant though.